tasks/connectors/lacework/lacework.rb

```diff
@@ -71,6 +74,8 @@ def run(opts)
         @kenna_connector_id = @options[:kenna_connector_id]
         @kenna_api_host = @options[:kenna_api_host]
 
+        @ratequeue = Limiter::RateQueue.new(680, interval: 3600, balanced: true)
```

Comment:
It would be nice to have a comment here on what the Limiter values are for.  680?  3600?